firefoxThemes/waifus/Aqua/theme.json

```diff
@@ -135,12 +135,7 @@
       251,
       251
     ],
-    "toolbar_field_separator": [
```

Comment:
Probably a good idea to remove this from the `firefox.theme.template.json`, because next time I run a `buildThemes` it is going to put it right back.

Reply:
As I am still a bit unfamiliar with the structure of the asset building, there is a *Kanna* light/dark background which is the same how does one remove it from the build tool?

Reply:
> there is a Kanna primary/secondary background which is the same how does one remove it from the build tool?

That one is on me. The background get copied over from my asset repository during the `buildThemes` process. I can modify and test things out, because I probably am not going to make a different secondary Kanna background.

Reply:
I see.